cashu/lightning/base.py

```diff
@@ -33,36 +34,79 @@ class InvoiceResponse(BaseModel):
     payment_request: Optional[str] = None
     error_message: Optional[str] = None
 
+class PaymentResult(Enum):
+    FAILED = 0
+    SETTLED = 1
+    PENDING = 2
+
+    UNKNOWN = 3
+
+    def __str__(self):
+        return self.name
+
+    # We assume `None` is `FAILED`
```

Comment:
I think that `None` should be `PENDING`, no?

Reply:
Maybe you are right. But we also used `None` to signal what now would be `UNKNOWN`

Reply:
Shouldn't matter too much, though. One shouldn't be using that. I only ever created it to use it in `FakeWallet`

Reply:
Then I suggest to change this code to assume that None is PENDING.

That way it is consistent with rest of the old and new code.